config/main.py

```diff
@@ -1093,7 +1114,9 @@ def remove(ctx, interface_name, ip_addr):
 
     if_table = ""
     try:
-        ipaddress.ip_network(unicode(ip_addr), strict=False)
+        ip_network = ipaddress.ip_network(unicode(ip_addr), strict=False)
+        if not interface_name.startswith("Loopback") and ip_network.prefixlen == ip_network.max_prefixlen:
+            ctx.fail("Bad mask /{} for 'ip_addr'".format(ip_network.max_prefixlen))
```

Comment:
Suggest printing the IP address in the failure message.

E.g., `ctx.fail("Bad mask /{} for IP address {}".format(ip_network.max_prefixlen, ip_addr))`

Reply:
Done